plugin/db/mysql/mysql.go

```diff
@@ -236,12 +236,10 @@ func (driver *Driver) SyncSchema(ctx context.Context) ([]*db.User, []*db.Schema,
 		}
 
 		key := fmt.Sprintf("%s/%s", dbName, tableName)
-		indexList, ok := indexMap[key]
-		if ok {
+		if indexList, ok := indexMap[key]; ok {
 			indexMap[key] = append(indexList, index)
 		} else {
-			list := make([]db.Index, 0)
-			indexMap[key] = append(list, index)
+			indexMap[key] = append([]db.Index(nil), index)
```

Comment:
indexMap[key] = []db.Index{index} ? and below

Reply:
Fixed.